# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1380967981 -7200
#      Sat Oct 05 12:13:01 2013 +0200
# Node ID 387ecd448b30d69498bfc6fba3ef92ac8e94ffcb
# Parent  28b5f3f0242658329de96b8410ed982f94584c4a
main-windows.cc: Fixed typo in open_developer_page URL

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -268,17 +268,17 @@ void
 main_window::open_contribute_page (void)
 {
   QDesktopServices::openUrl (QUrl ("http://octave.org/donate.html"));
 }
 
 void
 main_window::open_developer_page (void)
 {
-  QDesktopServices::openUrl (QUrl ("http://ocxtave.org/get-involved.html"));
+  QDesktopServices::openUrl (QUrl ("http://octave.org/get-involved.html"));
 }
 
 void
 main_window::process_settings_dialog_request (void)
 {
   settings_dialog *settingsDialog = new settings_dialog (this);
   int change_settings = settingsDialog->exec ();
   if (change_settings == QDialog::Accepted)
